Public/Src/FrontEnd/Nuget/NugetResolver.cs

```diff
@@ -20,6 +25,7 @@ namespace BuildXL.FrontEnd.Script
     /// </summary>
     public sealed class NugetResolver : DScriptSourceResolver
     {
+        internal const string NugetResolverName = "ComponentGovernance";

```

Comment:
>ComponentGovernance [](start = 51, length = 19)

Nit - I'd call this manifest generator or something since we aren't actually performing the CG checks here. I wouldn't want to mislead anyone

Reply:
And the const member's name too

---
In reply to: [325449958](https://github.com/microsoft/BuildXL/pull/889#discussion_r325449958) [](ancestors = 325449958)